viewer/packages/camera-manager/src/Flexible/FlexibleControls.ts

```diff
@@ -129,6 +131,18 @@ export class FlexibleControls extends EventDispatcher<FlexibleControlsEvent> {
     return this._options;
   }
 
+  get camera(): PerspectiveCamera | OrthographicCamera {
```

Comment:
We do not have support for `OrthographicCamera`, it is not necessary to return `OrthographicCamera` type

Reply:
Just for the future